src/Core/Checkout/Payment/PaymentProcessor.php

```diff
@@ -180,8 +225,13 @@ private function getCurrentOrderTransaction(string $orderId, Context $context):
         return $transaction;
     }
 
-    private function getToken(OrderTransactionEntity $transaction, ?string $finishUrl, ?string $errorUrl, SalesChannelContext $salesChannelContext): string
+    private function getOldToken(OrderTransactionEntity $transaction, ?string $finishUrl, ?string $errorUrl, SalesChannelContext $salesChannelContext): string
     {
+        if (!$this->tokenFactory) {
+            // @phpstan-ignore-next-line
```

Comment:
would you mind changing it to this? 
```suggestion
            // @phpstan-ignore specific.identifier.change.me! (comment why we need to ignore it)
```

also see https://phpstan.org/user-guide/ignoring-errors
ignore-next-line will ignore every PHPStan issue and not just the specific one.

Reply:
was unnecessary anyway ("vorauseilender Gehorsam"). I assumed there would be an issue because I'm throwing a non-domain-exception, but PHPstan doesn't complain